src/components/AnalysisPage/ThicknessAnalysisBox/ThickAnalysisBox.jsx

```diff
@@ -45,21 +45,21 @@ const ThickAnalysisBox = ({
   const getSpecificDataRegardingThreshold = (value) => {
     //get the data from listOfThicknessData that matches the threshold value and units value
     const data = listOfThicknessData.find(
-      (data) => data.threshold === value && data.units === units //**added units check
+      (data) => Number(data.threshold) == Number(value) && data.units === units //**added units check
     );
```

Comment:
is `Number(data.threshold)` converting what potentially is a string, back to a number?

Reply:
Yea I think that data.threshold might always be a number whereas `value` is potentially a string but converting them both to numbers if just to be safe I guess.

And I want them both to be numbers for this comparison because if they are potentially a string then something like ".7" will not equal "0.7" but when they are both converted to numbers then they will be equal